.github/workflows/test.yml

```diff
@@ -0,0 +1,23 @@
+name: CI
+
+on:
+  pull_request:
+    branches:
+      - master
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    name: Install dependencies
```

Comment:
The custom job name doesn't describe the job, only one of its steps. Copy/paste error?

```suggestion
```